MySelenium/src/test/java/steps/NavigatorSteps.java

```diff
@@ -0,0 +1,18 @@
+package steps;
+
+import driver.Driver;
+import org.testng.Assert;
+import pages.NavigatorPage;
+
+
+public class NavigatorSteps extends BaseSteps{
+    NavigatorPage navigatorPage = new NavigatorPage(driver);
+
+    public NavigatorSteps(Driver driver){
+        super(driver);
+    }
+
+    public void textEqualTopicList (String findElement) {
+        Assert.assertTrue(navigatorPage.getTopicSearchResults().stream().allMatch(x->x.toLowerCase().contains(findElement.toLowerCase())));
```

Comment:
use hamcrest for assertions. watch video or read documentation. after adding harmrest u can create own assert method if it does not have such